### ISSUE TITLE
HTML5 validation for the control

### ISSUE THREAD

**Issue by akoval** [powmedia/backbone-forms#499]
Numeric input control can be invalid (for example` <input type=number>` with **123....456** text).
but validation mechanism returns OK.

It uses `this.$el.val()` which returns empty for invalid control. As a result all validators passed (except required validator). So for optional fields form will be always committed to server with incorrect input.

Maybe need to add some checking to verify control state?
something like:
`this.$el.is(':invalid')`

**Comment by glenpike** [comment]
This seems more like a roadmap thing to consider - making validation work with HTML5 elements.  We should probably discuss this @exussum12 @powmedia ?

**Comment by exussum12** [comment]
Agreed, The support for these elements is pretty small currently. This should be supported though as the usage will rise

**Comment by powmedia** [comment]
I've been using HTML5 validation more and more, it could be a good idea to
at least use the same API and polyfill where required?  Could use existing
libraries for that.

**Comment by philfreo** [comment]
The support is actually very good these days (94% global support):
http://caniuse.com/#search=validation
